docs/visualize.asciidoc

```diff
@@ -1,2 +1,131 @@
 [[visualize]]
-== Visualizing your Data
\ No newline at end of file
+== Visualize
+
+You can use Kibana's _Visualize_ app to design and create data visualizations. You can save these visualizations, use
```

Comment:
Can we avoid referring to the different pages as "apps"? I think this is unnecessarily confusing. Kibana is the app, the Visualize page is an element of that app.